### PR TITLE
Added option to the run distributed task to consider standalone dta agent

### PR DESCRIPTION
**Pull request by cltshivash**
Added option to the run distributed task to consider standalone dta agent

### CODE REVIEW

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 [hunk 1]
```diff
@@ -68,6 +69,14 @@ foreach ($suite in $suites)
     }    
 }
 
+$testMachinesGroupName=$testMachineGroup
+if ([string]::Equals($env:UseDtaV2Agent, "true", [System.StringComparison]::OrdinalIgnoreCase)) {
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: >UseDtaV2Agent [](start = 26, length = 13)

guess this works when no variable is defined?

**Comment by acesiddhu** [comment]
:shipit:

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 (hunk 1 above)
Comment (cltshivash) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: Yes it does

---
In reply to: [104138653](https://github.com/Microsoft/vsts-tasks/pull/3727#discussion_r104138653) [](ancestors = 104138653)

File: Tasks/RunDistributedTests/RunDistributedTests.ps1 [hunk 2]
```diff
@@ -68,6 +69,14 @@ foreach ($suite in $suites)
...
+    # dtav2://env/TestProject/_apis/testrig/<name>
+    $testMachinesGroupName=[string]::Format("dtav2://env/{0}/_apis/testrig/{1}", $env:SYSTEM_TEAMPROJECT, $testMachineGroup.ToLowerInvariant())
```
Comment (vimegh) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: >$testMachineGroup [](start = 106, length = 17)

Can this be ever null?

Comment (vimegh) [inline Tasks/RunDistributedTests/RunDistributedTests.ps1]: talked with Shiva. Its cant be null. 

---
In reply to: [104144375](https://github.com/Microsoft/vsts-tasks/pull/3727#discussion_r104144375) [](ancestors = 104144375)

**Comment by vimegh** [comment]
:shipit:

**Pull request merged**